client/webserver/site/src/js/markets.js

```diff
@@ -1677,7 +1677,7 @@ export default class MarketsPage extends BasePage {
     }
     const lotSize = this.market.cfg.lotsize
     page.lotField.value = lots
-    page.qtyField.value = Doc.formatCoinValue(lots * lotSize, this.market.baseUnitInfo)
+    page.qtyField.value = convertToConventional(lots * lotSize, this.market.baseUnitInfo.conventional.conversionFactor)
```

Comment:
Seems like the function is unnecessary. It's just division.

Reply:
Will remove.

Reply:
Removed.